src/packages/@workspaces/features/socket-explorer/store/websocket-listener.ts

```diff
@@ -0,0 +1,4 @@
+import type { UnlistenFn } from "@tauri-apps/api/event";
+import { writable } from "svelte/store";
+
+export const webSocketListener = writable<Map<string, UnlistenFn>>(new Map());
```

Comment:
no need to create a new store, already handling  a hashmap based store for web socket with corresponding to tabId. that holds tab related data at one place